exist-core/src/main/java/org/exist/xquery/functions/fn/JSON.java

```diff
@@ -90,7 +90,7 @@ public class JSON extends BasicFunction {
     static final FunctionSignature[] FS_JSON_TO_XML = functionSignatures(
             FS_JSON_TO_XML_NAME,
             "Parses a string supplied in the form of a JSON text, returning the results in the form of an XML document node.",
-            returnsOpt(Type.ITEM, "The parsed data as XML"),
+            returnsOpt(Type.DOCUMENT, "The parsed data as XML"),
```

Comment:
any idea bout side-effects?

Reply:
@dizzzz How do you mean? It seems the function was previously alwats returning a Document or empty sequence, and that it was just the signature that was incorrect.